<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gui</class>
 <widget class="QWidget" name="gui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>322</width>
    <height>325</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>gui</string>
  </property>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>281</width>
     <height>201</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="manufacturerLineEdit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>240</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="totalLineEdit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>270</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="totalButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>112</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Show cars</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Manufacturer name:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
